web/js/components/layer/product-picker/browse/browse-layers.js

```diff
@@ -66,6 +68,8 @@ function BrowseLayers (props) {
     && selectedProjection === 'geographic'
     && categoryType !== 'recent';
 
+  const selectedCategoryName = lastAction.type === 'PRODUCT_PICKER/TOGGLE_CATEGORY_MODE' ? '' : category;
```

Comment:
This will work but, instead of having to look at the last action that was dispatched, I think we'd be better off setting the value itself in the Redux store when this action is dispatched.   So, in order to do that, you would just need to clear the value of `state.productPicker.category` when the `TOGGLE_CATEGORY_MODE` action is dispatched.  

You can do this by updating the reducer for that action to set this value along with what's already happening in there. For example right now it looks like:

```
case TOGGLE_CATEGORY_MODE: {
      return {
        ...state,
        mode: 'category',
        selectedLayer: null,
        selectedMeasurement: null,
        selectedMeasurementSourceIndex: 0,
      };
    }
```

And could be updated to:
```
case TOGGLE_CATEGORY_MODE: {
      return {
        ...state,
        mode: 'category',
        category: null
        selectedLayer: null,
        selectedMeasurement: null,
        selectedMeasurementSourceIndex: 0,
      };
    }
```

Reply:
Resolved in https://github.com/nasa-gibs/worldview/pull/3783/commits/31bbe83ca76fa35171306e3253d9c8efcccb6a87